bids-validator/validators/json/schemas/dataset_description.json

```
@@ -52,6 +52,19 @@
     "DatasetDOI": {
       "type": "string"
     },
+    "DatasetLinks": {
+      "type": "object",
+      "properties": { },
+      "propertyNames": {
+        "type": "string",
+        "minLength": 1,
+        "pattern": "^[a-zA-Z0-9]*$"
+      },
+      "additionalProperties": {
+        "type": "string",
+        "format": "uri"
```

Apparently what we actually want is a [uri-reference](https://json-schema.org/draft/2019-09/json-schema-validation.html#rfc.section.7.3.5).

```suggestion
        "format": "uri-reference"
```